public_html/quickstart.php

```diff
@@ -511,7 +513,8 @@
 				</div>
 				<div class="guide-tx1-heading darkmode-txt">
 					<p>
-						 Your title is now decrypted.
+						Using 3k3y IsoTools press <span class="txt-highlight darkmode-highlight">Decrypt</span> button and select the .iso with patched-in decryption keys. This will produce decrypted .dec.iso file.<br>

```

Comment:
since we seem to be highlighting extensions the first time they're mentioned, maybe block .dec.iso